docs/spelling_wordlist.txt

```diff
@@ -55,6 +55,8 @@ elasticsearch7
 enqueue
 entrypoint
 entrypoints
+env
+Env
```

Comment:
does the case matter on these?

Reply:
I think I was told "yes" by someone, but I can't entirely remember. I can try without.